### PR TITLE
feat(notes): adding message note for transfer action dialog

### PR DESCRIPTION
**Pull request by OrelKaravani**

### CODE REVIEW

**Review by RoeiRom — DISMISSED** [review]

File: client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTableFooter/TransferInvestigationsDialogs/TransferInvestigationDialogNote.tsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+import { Typography } from '@material-ui/core';
```
Comment (ItayBenMoshe) [inline client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTableFooter/TransferInvestigationsDialogs/TransferInvestigationDialogNote.tsx:3]: should be in the first section with react

File: client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTableFooter/TransferInvestigationsDialogs/TransferInvestigationInvestigator.tsx [hunk 2]
```diff
@@ -90,6 +91,8 @@ const TransferInvestigationDesk = (props: Props) => {
                             }
                         />
                     </div>
+                        <TransferInvestigationDialogNote />
+
```
Comment (ItayBenMoshe) [inline client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTableFooter/TransferInvestigationsDialogs/TransferInvestigationInvestigator.tsx:95]: delete row

**Review by ItayBenMoshe — CHANGES_REQUESTED** [review]

File: client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTableFooter/TransferInvestigationsDialogs/TransferInvestigationInvestigator.tsx (hunk 2 above)
Comment (OrelKaravani) [reply client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTableFooter/TransferInvestigationsDialogs/TransferInvestigationInvestigator.tsx]: fixed

File: client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTableFooter/TransferInvestigationsDialogs/TransferInvestigationDialogNote.tsx (hunk 1 above)
Comment (OrelKaravani) [reply client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTableFooter/TransferInvestigationsDialogs/TransferInvestigationDialogNote.tsx:2]: Fixed

**Review by RoeiRom — APPROVED** [review]

**Review by ItayBenMoshe — APPROVED** [review]

**Pull request merged**